circuits-circom/regexes/hdfc/hdfc_accnum.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parts": [
+    {
+      "is_public": false,
+      "regex_def": "\\*\\*"
```

Comment:
Maybe add a line break \r\n to make it a bit safer

Reply:
There wouldn't always be a line break.